examples/deploy/addon/helloworld/resources/helloworld_clustermanagementaddon.yaml

```diff
@@ -2,10 +2,22 @@ apiVersion: addon.open-cluster-management.io/v1alpha1
 kind: ClusterManagementAddOn
 metadata:
   name: helloworld
+  annotations:
+    "addon.open-cluster-management.io/lifecycle": "addon-manager"
 spec:
   addOnMeta:
     displayName: helloworld
     description: "helloworld is an example addon created by go template"
   supportedConfigs:
   - group: addon.open-cluster-management.io
     resource: addondeploymentconfigs
+  installStrategy:
+    type: Placements
+    placements:
+      - name: global
+        namespace: default
```

Comment:
Might need to use namespace open-cluster-management, since examples/deploy/addon/helloworld/kustomization.yaml will override the namespace.